module/VuFind/tests/unit-tests/src/VuFindTest/Search/Solr/ResultsTest.php

```diff
@@ -59,6 +60,60 @@ public function testCursorMark(): void
         $this->assertEquals('foo', $results->getCursorMark());
     }
 
+    /**
+     * Test facet translation functionality.
+     *
+     * @return void
+     */
+    public function testFacetTranslations(): void
+    {
+        $mockTranslator = $this->createMock(TranslatorInterface::class);
+        $mockTranslator->expects($this->exactly(2))
```

Comment:
@mtrojan-ub, I just made a couple of commits to make this part of the test more specific and more realistic; the mock was prematurely producing the final expected value in a previous revision, but now it should be working more like it's supposed to. :-)